clouddriver-api/clouddriver-api.gradle

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+dependencies {
+  implementation platform("com.netflix.spinnaker.kork:kork-bom:$korkVersion")
+  annotationProcessor platform("com.netflix.spinnaker.kork:kork-bom:$korkVersion")
+
+  api("com.netflix.spinnaker.kork:kork-plugins-api")
+
+  implementation "com.netflix.spinnaker.moniker:moniker"
```

Comment:
It's nice that this is implementation and it looks like moniker is pretty thin and not too scary, dependency wise.  This may be ok.

Reply:
Rob Z said this one was acceptable.